tests/test_filter_count.py

```diff
@@ -779,6 +779,26 @@ def test_num_label_issues(confident_joint):
             )
 
 
+@pytest.mark.parametrize("confident_joint", [None, True])
+def test_num_label_issues_multilabel(confident_joint):
+    dataset = multilabel_data
+    n = count.num_label_issues(
+        labels=dataset["labels"],
+        pred_probs=dataset["pred_probs"],
+        confident_joint=dataset["cj"] if confident_joint else None,
+        estimation_method="off_diagonal",
+        multi_label=True,
+    )
+    f = filter.find_label_issues(
+        labels=dataset["labels"],
+        pred_probs=dataset["pred_probs"],
+        confident_joint=dataset["cj"] if confident_joint else None,
+        multi_label=True,
+    )
+    assert sum(f) == 53
+    assert sum(f) == n
```

Comment:
```suggestion
    assert sum(f) == n
```

Reply:
don't think we need to test filter.find_label_issues here too?

Reply:
Fixed: https://github.com/aditya1503/cleanlab/blob/a684583aa0e89de6414172c891b1db305e31ade3/tests/test_filter_count.py#L798